src/thunderbots/software/param_server/cfg/Params.cfg

```diff
@@ -25,17 +28,18 @@ navigator_params = gen.add_group("Navigator")
 
 navigator_params.add("default_avoid_dist", double_t, 0, "brief description", 50,  0, 100)
 navigator_params.add("collision_avoid_velocity_scale", double_t, 0, "brief description", .5, 0,  1)
-#add more navigator parameters here
+# add more navigator parameters here
 
 #######################################################################
 #                              EXAMPLES                               #
 #######################################################################
 
-example_params = gen.add_group("Examples")
+#example_params_parent = gen.add_group("Examples parent")
+#example_params = example_params_parent.add_group("Examples child")
 
-example_params.add("string_example", str_t, 0,"An example string parameter", "Hello World")
-example_params.add("int_example", int_t, 0, "An example int parameter", 50,  0, 100)
-example_params.add("double_example", double_t, 0, "An example double parameter", 0.4,  0, 100)
-example_params.add("bool_example", bool_t, 0, "An example bool parameter", True)
+#example_params.add("string_example", str_t, 0,"An example string parameter", "Hello World")
```

Comment:
super nit; I think this would be a bit easier to read if there was a space between the `#` and the code
`# example_params.add(...`

Reply:
fixed